packages/fxa-customs-server/lib/user_defined_rules.js

```diff
@@ -60,5 +55,23 @@ module.exports = function (config, fetchRecord, setRecords) {
     }
 
     return result;
+  }
+
+  async function checkUserDefinedRateLimitRulesByIpEmail(
+    result,
+    action,
+    email,
+    ip
+  ) {
+    return checkUserDefinedRateLimitRulesByKey(result, action, ip + email);
```

Comment:
I'm not sure it matters, expect for existing customs count, but I don't _think_ this produces exactly the same hash as before, which was called with `utils.createHashHex(email,ip)`

Reply:
Ah good find, I'll  update to pass in as arguments.